src/scenarioLoading.ts

```diff
@@ -46,11 +46,15 @@ export const loadAllScenarios = (
                     )}-${_i}`;
                     const ctx = { scenario: scenarioNameWithIdx };
 
-                    const actionCatalogWithReplacedLoadVariables: Action[] = [];
+                    const actionCatalogWithReplacedLoadVariables: Action[] = JSON.parse(
```

Comment:
What is the purpose of stringifying and parsing?

Reply:
deep copy ;)